### PR TITLE
Added system-probe installation using

### PR DESCRIPTION
**Pull request by guyarb**
Added an option to install system-probe (using default configuration) via environment variable `DD_SYSTEM_PROBE_ENSURE_CONFIG=true`

### CODE REVIEW

File: test/localtest.sh [hunk 1]
```diff
@@ -79,4 +79,28 @@ if [ -n "${EXPECTED_TOOL_VERSION}" ]; then
     fi
 fi
 
+system_probe_config_file=/etc/datadog-agent/system-probe.yaml
+if [ -n "${DD_SYSTEM_PROBE_ENABLED}" ]; then
+    if [ -e "$system_probe_config_file" ]; then
+        echo "[OK] Found system-probe configuration file $system_probe_config_file"
```
Comment (val06) [inline test/localtest.sh:85]: use here shell colors as well, green for OK, red for FAIL

**Review by val06 — CHANGES_REQUESTED** [review]
cred

File: test/localtest.sh (hunk 1 above)
Comment (guyarb) [reply test/localtest.sh:85]: that's not being used in the entire script, I don't think it is in the scope of the PR

**Review by val06 — APPROVED** [review]

**Comment by bkabrda** [comment]
So the variable is called `DD_SYSTEM_PROBE_ENABLED` but the service is not actually started/enabled. However, by default it can't really meaningfully start it because the example config is blank.

I think we should either still start/enable the service and inform the user that it might not actually run if the config file is empty; or we can rename the variable to be more descriptive and inform the user that it will just set up the config file. I can discuss with our team, but in the meanwhile I'd also like to think what your opinion is @guyarb. Thanks!

**Comment by guyarb** [comment]
> So the variable is called `DD_SYSTEM_PROBE_ENABLED` but the service is not actually started/enabled. However, by default it can't really meaningfully start it because the example config is blank.
> 
> I think we should either still start/enable the service and inform the user that it might not actually run if the config file is empty; or we can rename the variable to be more descriptive and inform the user that it will just set up the config file. I can discuss with our team, but in the meanwhile I'd also like to think what your opinion is @guyarb. Thanks!

Hey @bkabrda 
first of all, I added USM to system-probe.yaml.examle in https://github.com/DataDog/datadog-agent/pull/14908/files, but when I ran the localtest today, and looked into system-probe.yaml.examle, i didn't see we have usm there. Did I miss anything in the process?

Regarding your point, even if I manually adding some content to `system-probe.yaml` and run `service datadog-agent start` I still do not see system probe running
What am I missing in the process?
We can add the log as you suggested

File: install_script.sh.template [hunk 2]
```diff
@@ -806,6 +812,20 @@ if [ -n "$fips_mode" ]; then
...
+# set the system-probe configuration
+if [ -n "$system_probe_enabled" ]; then
+  if [ -e "$system_probe_config_file" ] && [ -z "$upgrade" ]; then
+    printf "\033[34m\n* Keeping old $system_probe_config_file configuration file\n\033[0m\n"
+  else
+    if [ ! -e "$system_probe_config_file" ]; then
+      $sudo_cmd cp "$system_probe_config_file.example" "$system_probe_config_file"
+    fi
+
+    $sudo_cmd chown dd-agent:dd-agent "$system_probe_config_file"
```
[the remark this answers is not in the document]
Comment (guyarb) [reply install_script.sh.template:824]: made the change, thanks

**Comment by guyarb** [comment]
@bkabrda bump

**Comment by guyarb** [comment]
@bkabrda I addressed your comment, let me know if I'm missing anything

**Review by bkabrda — APPROVED** [review]

**Pull request merged**